src/player.ts

```diff
@@ -422,6 +426,8 @@ export default class Player extends EventEmitter {
     }
 
     this.data.power.gaia = 0;
+    this.data.gaiaformers += this.data.gaiaformersInGaia;
```

Comment:
I don't think it should be there.

Reply:
It's not decreased in `gainReward`, and `hasResource` also expects it to not be decreased